help/en/package/jmri/jmrit/withrottle/Protocol.shtml

```diff
@@ -1037,7 +1039,7 @@ PFT1550686525&lt;;&gt;4.0
 
     <h3>Observations</h3>
 
-    <p>The Digitrax LnWI sends multiple instances of the H command as part
+    <p>The Digitrax LnWi sends multiple instances of the H command as part
```

Comment:
LNWI should be all caps.

Reply:
@rhwood do you have a reference for that? In my conversations with AJ at Digitrax, he always used "LnWi". I see both in use on their website.